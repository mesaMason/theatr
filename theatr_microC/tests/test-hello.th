int main()
{
    printstr("Hello World");
    printstr("hello world!");
    printstr("hello world hello  world   ");
    print(2);
    return 0;
}
