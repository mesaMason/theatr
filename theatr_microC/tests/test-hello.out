Hello World
hello world!
hello world hello  world 
2
